ContactUsBottom Component #35

ssinikoglu opened the pull request:
This PR consists of ContactUsBottom component of Contact page. The component has only visual features (bottom curve which is similar to curve in Home page and images placed in curve) and responsive-friendly. 

Last PR for the Contact Page will be Contact Container.

Please review. I am looking forward to your comments.

ssinikoglu committed:
Feat: create component files

ssinikoglu committed:
Test: create snapshot test

ssinikoglu committed:
Style: add primary styling

ssinikoglu committed:
Feat: add an image

ssinikoglu committed:
Perf: add bottom curve

ssinikoglu committed:
Merge branch 'dev' into contactUsBottom

ssinikoglu committed:
Refactor: fix lines and spaces

ssinikoglu committed:
Style: add initial positioning

ssinikoglu committed:
Refactor: update class names

ssinikoglu committed:
Style: add pc view styling

ssinikoglu committed:
Style: add media breakpoints

ssinikoglu committed:
Refactor: add notes about breakpoints

ssinikoglu committed:
Merge branch 'dev' into contactUsBottom

ssinikoglu committed:
Test: run snapshot test

ssinikoglu committed:
Refactor: run yarn fix

ssinikoglu committed:
Style: change class names

ssinikoglu committed:
Test: rerun test

ssinikoglu committed:
Merge branch 'dev' into contactUsBottom

ssinikoglu committed:
Merge branch 'dev' into contactUsBottom

ssinikoglu committed:
Refactor: update class names

ssinikoglu committed:
Style: change position adjustments

ssinikoglu committed:
Style: update responsive scales

ssinikoglu committed:
Refactor: edit class names

ssinikoglu committed:
Merge branch 'dev' into contactUsBottom

ssinikoglu committed:
Style: fix responsive view

ssinikoglu committed:
Refactor: run yarn fix

ssinikoglu committed:
Test: rerun test after changes

ssinikoglu committed:
Refactor: trim final new lines

ssinikoglu committed:
Refactor: trim final new line

ssinikoglu committed:
Refactor: change new line settings

ssinikoglu committed:
Refactor: run yarn fix

ssinikoglu committed:
Merge branch 'dev' into contactUsBottom

ssinikoglu committed:
Style: update media query notes

louisrli commented on .vscode/settings.json:
```diff
@@ -0,0 +1,4 @@
+{
+	"files.trimFinalNewlines": true,
```
proofread pr

ssinikoglu replied:
After run yarn fix my code editor was adding new lines, that's why I have changed it. But for team, can it be irrelevant? If it is, I can delete it and only change my user settings.

louisrli replied:
still needs to be removed even after adding .gitignore. also theres a typo in the gitignore

ssinikoglu replied:
Thanks a lot for your help Louis, I removed the file and fixed the typo.

louisrli commented on src/components/ContactUsBottom/index.jsx:
```diff
@@ -0,0 +1,39 @@
+
+            <img
+              className="contactUsBottomContainer__bottomCurve__topHalfImages--subImgOne"
+              alt="girl playing in the garden"
```
capitalize. Girl playing in the garden

louisrli replied:
and all else

ssinikoglu replied:
Thanks for reviewing. All alt values are capitalized.

louisrli commented:
use .gitignore

On Tue, Nov 17, 2020 at 11:10 PM Sule Sinikoglu <notifications@github.com>
wrote:

> *@ssinikoglu* commented on this pull request.
> ------------------------------
>
> In .vscode/settings.json
> <https://github.com/ReCoded-Org/capstone-istanbul-juniper/pull/70#discussion_r525477792>
> :
>
> > @@ -0,0 +1,4 @@
> +{
> +	"files.trimFinalNewlines": true,
>
> After run yarn fix my code editor was adding new lines, that's why I have
> changed it. But for team, can it be irrelevant? If it is, I can delete it
> and only change my user settings.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/ReCoded-Org/capstone-istanbul-juniper/pull/70#discussion_r525477792>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJ6G2PNRYPJP7DOA5WTMRTSQLKETANCNFSM4TZAOV5Q>
> .
>

ssinikoglu committed:
Refactor: move to .gitignore

"vscode.settings.json" file is moved  to .gitignore.

ssinikoglu committed:
Refactor: update alt attribute

ssinikoglu committed:
Test: rerun test after updates

ssinikoglu committed:
Merge branch 'dev' into contactUsBottom

ssinikoglu commented:
> use .gitignore
> […](#)
> On Tue, Nov 17, 2020 at 11:10 PM Sule Sinikoglu ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In .vscode/settings.json <[#70 (comment)](https://github.com/ReCoded-Org/capstone-istanbul-juniper/pull/70#discussion_r525477792)> : > @@ -0,0 +1,4 @@ +{ + "files.trimFinalNewlines": true, After run yarn fix my code editor was adding new lines, that's why I have changed it. But for team, can it be irrelevant? If it is, I can delete it and only change my user settings. — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub <[#70 (comment)](https://github.com/ReCoded-Org/capstone-istanbul-juniper/pull/70#discussion_r525477792)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAJ6G2PNRYPJP7DOA5WTMRTSQLKETANCNFSM4TZAOV5Q> .

Thanks for your help. As you have said, I added vscode settings.json to .gitignore.

ssinikoglu committed:
Fix: add vscode settings to gitignore

majdajroudi commented on src/components/ContactUsBottom/index.jsx:
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import contactUsBottomImgRight_1 from "../../images/child.png";
+import contactUsBottomImgRight_2 from "../../images/child_2.png";
```
contactUsBottomImgRight_0
contactUsBottomImgRight_1

ssinikoglu replied:
Thanks for reviewing Majd, I have changed names according to your comment.

majdajroudi commented on src/components/ContactUsBottom/index.jsx:
```diff
@@ -0,0 +1,39 @@
+import contactUsBottomImgRight_2 from "../../images/child_2.png";
+import contactUsBottomImgLeft from "../../images/family.png";
+import "./index.css";
+import "antd/dist/antd.css";
```
I think no need for this since it's already imported in App.jsx

ssinikoglu replied:
Thanks, you are absolutely right! I deleted the import.

majdajroudi reviewed (approved):
minor comments, but good work!

ssinikoglu committed:
Refactor: update image names

ssinikoglu committed:
Fix: fix typo at gitignore

ssinikoglu committed:
Refactor: delete unneccassary import

ssinikoglu committed:
Refactor: run yarn fix

ssinikoglu commented:
PR has been approved @Ammar-64  PTL

Ammar-64 committed:
Merge branch 'dev' into contactUsBottom
